examples/tensorflow_hello_world/helloworld.py

```diff
@@ -1,11 +1,211 @@
+import math
+
 import tensorflow as tf
+from absl import app, flags
+from tensorflow.keras.layers import Conv2D, Dense, Flatten, Input, MaxPool2D
+
+from xain.datasets import load_splits
+from xain.fl.coordinator import Coordinator, RandomController
+from xain.fl.coordinator.aggregate import FederatedAveragingAgg
+from xain.fl.participant import ModelProvider, Participant
+
+# TODO: this feels a bit hacky. it's the only flag that needs to be added here.
+#  any better solution?
+FLAGS = flags.FLAGS
+flags.DEFINE_string("task_name", None, "")
```

Comment:
This is not being used at a later point so we can remove it

Reply:
if i remove this, it complains the flag doesn't exist when writing summaries inside logging.py:

`Traceback (most recent call last):
  File "examples/tensorflow_hello_world/helloworld.py", line 247, in <module>
    app.run(main=main)
  File "/Users/anselmosampietro/Documents/XAIN/main37/lib/python3.7/site-packages/absl/app.py", line 300, in run
    _run_main(main, args)
  File "/Users/anselmosampietro/Documents/XAIN/main37/lib/python3.7/site-packages/absl/app.py", line 251, in _run_main
    sys.exit(main(argv))
  File "examples/tensorflow_hello_world/helloworld.py", line 80, in main
    validation_metrics, _, _, _ = coordinator.fit(num_rounds=R)
  File "/Users/anselmosampietro/Documents/XAIN/xain/xain/fl/coordinator/coordinator.py", line 82, in fit
    train_round=r,
  File "/Users/anselmosampietro/Documents/XAIN/xain/xain/fl/logging/logging.py", line 47, in write_summaries
    tag=f"coordinator_{FLAGS.task_name}/val_acc", simple_value=val_acc
  File "/Users/anselmosampietro/Documents/XAIN/main37/lib/python3.7/site-packages/absl/flags/_flagvalues.py", line 473, in __getattr__
    raise AttributeError(name)
AttributeError: task_name`